### DISCUSSION TITLE
Website unreachable

### DISCUSSION THREAD

**Discussion by emrade** [code-market-software/code.market#5] [General]
The website has been down for at least 2 days now..
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/25217766/202845674-ea638882-d5e8-40f0-bc98-6e5003f89c13.png">

**Comment by emrade** [comment]
Its back up now.

**Reply by code-market-software** [reply]
@emrade sorry for the late reply. yes we had problem with hosting provider at this time.

**Reply by elgaxton** [reply]
This is message I'm receiving when I tried downloading a product: Application error: a client-side exception has occurred (see the browser console for more information).

**Reply by code-market-software** [reply]
Hello @elgaxton, I hope this message finds you well. We were wondering if you could confirm if the issue you mentioned is still present. If it is, could you kindly provide us with a link to the product and the order ID so that I may better assist you with resolving the issue? Thank you in advance for your help and cooperation.
